EngineeringModel/ViewModels/Dialogs/ActualFiniteStateListDialogViewModel.cs

```diff
@@ -48,6 +48,23 @@ public class ActualFiniteStateListDialogViewModel : CDP4CommonView.ActualFiniteS
         /// Backing field for <see cref="ExcludeOption"/>s
         /// </summary>
         private ReactiveList<Option> includeOption;
+        
+        /// <summary>
+        /// Backing field for <see cref="IsActualFiniteStatesListValid"/>
+        /// </summary>
+        private bool isActualFiniteStatesListValid;
+
+        /// <summary>
+        /// Gets a value indicating whether the ActualFiniteStateList needs to be re-calculated or not
```

Comment:
`Gets a value indicating whether the ActualFiniteStateList needs to be re-calculated or not`

becomes

`Gets a value indicating whether the ActualFiniteStates of this ActualFiniteStateList needs to be re-calculated or not`